cli/client_test.go

```diff
@@ -1050,3 +1050,42 @@ func TestShellFileCopy(t *testing.T) {
 		})
 	})
 }
+
+func TestReadOAuthApp(t *testing.T) {
+	readOAuthAppFunc := func(ctx context.Context, in *apppb.ReadOAuthAppRequest, opts ...grpc.CallOption) (
+		*apppb.ReadOAuthAppResponse, error,
+	) {
+		return &apppb.ReadOAuthAppResponse{
+			ClientName:   "clientname",
+			ClientSecret: "fakesecret",
+			OauthConfig: &apppb.OAuthConfig{
+				ClientAuthentication: apppb.ClientAuthentication_CLIENT_AUTHENTICATION_REQUIRED,
+				Pkce:                 apppb.PKCE_PKCE_REQUIRED,
+				UrlValidation:        apppb.URLValidation_URL_VALIDATION_ALLOW_WILDCARDS,
```

Comment:
could you add a test with some of the prefixes as well?

Reply:
Is this test not already testing the prefixes? These variables wouldn't pass otherwise (i.e. `CLIENT_AUTHENTICATION_REQUIRED` becoming `required`)